facts/3.14/redhat-7-x86_64.facts

```diff
@@ -210,30 +297,39 @@
         "noexec",
         "mode=755"
       ],
-      "size": "243.60 MiB",
-      "size_bytes": 255430656,
+      "size": "918.91 MiB",
+      "size_bytes": 963547136,
       "used": "0 bytes",
       "used_bytes": 0
+    },
+    "/vagrant": {
+      "available": "48.28 GiB",
+      "available_bytes": 51843301376,
+      "capacity": "30.99%",
+      "device": "vagrant",
+      "filesystem": "vboxsf",
+      "options": [
+        "rw",
+        "nodev",
+        "relatime"
+      ],
+      "size": "69.97 GiB",
+      "size_bytes": 75125227520,
+      "used": "21.68 GiB",
+      "used_bytes": 23281926144
     }
   },
   "mtu_eth0": 1500,
-  "mtu_eth1": 1500,
   "mtu_lo": 65536,
   "netmask": "255.255.255.0",
   "netmask6": "ffff:ffff:ffff:ffff::",
   "netmask6_eth0": "ffff:ffff:ffff:ffff::",
-  "netmask6_eth1": "ffff:ffff:ffff:ffff::",
-  "netmask6_lo": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
   "netmask_eth0": "255.255.255.0",
-  "netmask_eth1": "255.255.255.0",
   "netmask_lo": "255.0.0.0",
   "network": "10.0.2.0",
   "network6": "fe80::",
   "network6_eth0": "fe80::",
-  "network6_eth1": "fe80::",
-  "network6_lo": "::1",
```

Comment:
same here

Reply:
I don't see any facts that were missing previously and that you added, so maybe don't edit this factset at all?